api/clients/grailfiltersegements/filtersegments_test.go

```diff
@@ -0,0 +1,97 @@
+//go:build live
```

Comment:
Since it is not wired, I removed this file and will save it locally. When we're going to wire it, I'll add it. This produces more confusion than benefits.

This file is just a tiny wrapper that exposes basic HTTP operations, and it has no logic. For that reason, there is no need to complicate with unit tests